client/express-checkout/utils/index.js

```diff
@@ -256,17 +256,13 @@ const getRequiredFieldDataFromShortcodeCheckoutForm = ( data ) => {
 /**
  * Get array of payment method types to use with intent. Filtering out the method types not part of Express Checkout.
  *
+ * @see https://docs.stripe.com/elements/express-checkout-element/accept-a-payment#enable-payment-methods - lists the method types
+ * supported and which ones are required by each Express Checkout method.
+ *
  * @param {string} paymentMethodType Payment method type Stripe ID.
  * @return {Array} Array of payment method types to use with intent, for Express Checkout.
  */
 export const getExpressPaymentMethodTypes = ( paymentMethodType = null ) =>
 	getPaymentMethodTypes( paymentMethodType ).filter( ( type ) =>
-		[
-			'link',
-			'google_pay',
-			'apple_pay',
-			'paypal',
-			'amazon_pay',
-			'klarna',
```

Comment:
Previously, I had included `klarna` here as it will be available soon. But we can revisit this later when it happens.

Reply:
Yeah I figured that might be the case.

[Stripe docs](https://docs.stripe.com/elements/express-checkout-element/accept-a-payment) don't yet describe the `paymentMethodTypes` that would be needed for Klarna, but I suspect it would just be `klarna`. With that in mind, we probably wouldn't want to include it here yet in case Stripe flip the switch on their end and because we've preemptively added Klarna here, anyone who has Klarna already enabled would start seeing it automatically. 

When start supporting Klarna ECE, adding it here should be simple enough.